terraform/fec_rds.tf

```diff
@@ -105,6 +105,11 @@ resource "aws_db_parameter_group" "fec_default" {
         name  = "log_statement"
         value = "ddl"
     }
+
+    parameter {
+        name  = "max_standby_streaming_delay" # This has no effect on masters, it only affects slaves
+        value = "1 h"
```

Comment:
@vrajmohan, do you know if this is the correct way to set this?  Looking over the [Postgres documentation](https://www.postgresql.org/docs/9.6/static/runtime-config-replication.html#GUC-MAX-STANDBY-STREAMING-DELAY) appears to be a value in milliseconds, but it's also not entirely clear.  I'm trying to look in the RDS documentation too to see and verify.